argo/application-field-validation/artifacthub-pkg.yml

```diff
@@ -0,0 +1,21 @@
+name: application-field-validation
+version: 1.0.0
+displayName: application field validation
+createdAt: "2023-04-10T00:56:02.000Z"
+description: >-
+  This policy performs some best practices validation on Application fields. Path or chart must be specified but never both. And destination.name or destination.server must be specified but never both.
+install: |- # The installation instructions for the package
```

Comment:
No comments should be in any of these files.

Reply:
ok